content/400-pulse/100-what-is-pulse.mdx

```diff
@@ -6,23 +6,48 @@ tocDepth: 3
 toc: true
 ---
 
-<TopBlock>
+## What is Prisma Pulse?
 
-[Prisma Pulse](https://www.prisma.io/data-platform/pulse) is managed database-event infrastructure that captures and distributes your database events to your application. It simplifies subscribing to type-safe data changes with an extended [Prisma Client](/orm/prisma-client) to power real-time functionality.
+[Prisma Pulse](https://www.prisma.io/data-platform/pulse) is managed infrastructure that captures databases events (i.e. create, update and delete operations) and distributes them to your applications.
 
-</TopBlock>
+It lets you subsribe to any changes happening in your database using [Prisma Client](/orm/prisma-client) to power **real-time use cases** in your applications, such as:
 
-## How Prisma Pulse works
+- chat and messaging
+- data syncing (e.g. into a search index)
+- social interaction and collaboration
+- notifying users
+- updating inventories
+- managing payments
+- ... and any other functionality that requires real-time updates
 
-Prisma Pulse leverages Change Data Capture (CDC) to efficiently observe and capture database changes as they occur. By monitoring the database's transaction log, Prisma Pulse identifies change events like inserts, updates, and deletes without impacting the database's performance.
+Prisma Pulse integrates with [Prisma ORM](/orm) and easily lets you subscribe database changes in a type-safe way using Prisma Client:
 
-The captured events are processed, evaluated, and swiftly distributed to relevant client subscriptions ensuring your applications stay synchronized with the latest database state.
+![](/img/pulse/overview.png)
 
-This eliminates the need for complex polling or manual data synchronization, saving you development time and effort.
+## Why Prisma Pulse?
 
-![What is Pulse](/img/pulse/what-is-pulse.png)
+Building real-time functionality based on changes that occur in your database can be very complicated. Common approaches like _polling_, implementing _application-level updates_ or using _additional infrastructure_ (like Apache Kafka or RabbitMQ) either don't scale, are very costly or come with substantial development and maintenance overhead.
 
-## What you can build with Prisma Pulse
+Prisma Pulse is based on the idea of _unidirectional data flow_ which is implemented via [Change Data Capture](https://en.wikipedia.org/wiki/Change_data_capture) (CDC). CDC gets rid of the deficincies of the other approaches and solves the problem of reacting to database events in a robust and elegant manner.
```

Comment:
When we say “other approaches” what do we mean here?

Reply:
It's referencing the "Common approaches like _polling_, implementing _application-level updates_ or using _additional infrastructure_ (like Apache Kafka or RabbitMQ) either don't scale, are very costly or come with substantial development and maintenance overhead" from the first paragraph.

If that wasn't clear to you while reading, do you have an idea how this can be made clearer?